sync.py

```diff
@@ -72,12 +82,33 @@ def sync_directly(unsycned_builds):
         print("Removing " + build['nvr'] + ".src.rpm...")
         os.remove(build['nvr'] + ".src.rpm")
 
+local_packages = False
 # Badly written but working python script
 if __name__ == "__main__":
     # tag = sys.argv[1]
     tag = "rhel-8.2.0-candidate"
+    ctag = "dist-c8-stream-compose"
```

Comment:
I have not used this tag before. What does this mean ? Shall we add a comment about it ?

Reply:
I thought this was the normal c8 stream tag, we can just ping Brian about it today.